05/mobile.css

```diff
@@ -0,0 +1,203 @@
+@media (min-width: 10px) and (max-width: 800px){
+body{
+    max-width: 800px;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.all{
+    max-width: 800px;
+    max-height: 100vh;
+    
+}
+.header{
+    display: grid;
+    grid-template-columns: minmax(800px);
+    grid-template-rows: minmax(200px);
+    height: 99%;
+    border: solid;
+    border-color: green;  
+}
+
+.header__row{
+    display:grid;  
+    grid-template-columns: minmax(3, 150px 300px 300px);
+    grid-template-rows: minmax(200px);
+    border: solid;
+    border-color: rgb(31, 19, 207);
+    height: 80px;
+    grid-template-areas: 
+    "A B C D";
+}
+
+.header__element{
+    display: grid;
+    grid-area: B;
+    place-self: center;
+    width: 180px;
+    height: 40px;
+    background-color: yellow;
+}
+
+
+nav{
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    grid-area: D;
+}
+
+label{
+    place-self: center;
+    width: 50px;
+    height: 35px;
+    border: 0;
+    border-top: 5px solid rgb(11, 10, 10);
+    border-bottom: 3px;
+    background-color: transparent;
+    position: relative;
+}
+
+label::before, label::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    border-top: 5px solid  rgb(11, 10, 10);
+    transform: translateY(10px);
+}
+
+label::after{
+    transform: translateY(25px);
+}
+
+nav ul {
+    display: none; /* Ukrywamy menu początkowo */
+    position: absolute;
+    top: 100px;
+
+}
+
+nav ul li{
+    width: 100px;
+    height: 30px;
+    background-color: brown;
+}
+
+nav label:hover + input[type="checkbox"]:checked + ul {
+    display: block; /* Wyświetlamy menu po najechaniu na label i zaznaczeniu checkboxa */
+}
+  
+nav label:hover + input[type="checkbox"] {
+    display: none; /* Ukrywamy checkbox po najechaniu na label */
+}
```

Comment:
Checkbox ma być cały czas niewidoczny tj. `nav input {display: none;}`

PS. To dlatego menu skakało, bo checkbox był raz "widoczny" a raz nie. Piszę w cudzysłowie bo mozę i nie był on widoczny dla użytkownika, ale zajmował przestrzeń co powodowało przesuniacia innych elementów.